### PR TITLE
Add STAG rep role and add duties of Project lead

### PR DESCRIPTION
**Pull request by lumjjb**

### CODE REVIEW

**Review by TheFoxAtWork — APPROVED** [review]
LGTM!!

File: governance/roles.md [hunk 1]
```diff
@@ -140,6 +141,60 @@ Project Leads are nominated and approved by the following process:
 Depending on the expected length of the project, Project Leads may be assigned
 as OWNERS of the project directory in this repository to help facilitate and
  moderate new content to the project.
+
```
Comment (lumjjb) [inline governance/roles.md:144]:
```suggestion

```

File: governance/roles.md [hunk 2]
```diff
@@ -140,6 +141,60 @@ Project Leads are nominated and approved by the following process:
...
+- Ensure smooth day-to-day operations of the project. This includes:
+  - Ensuring meetings are scheduled, announced and recorded
+  - Facilitation of meetings, ensuring minutes are recorded
+- Ensure progress and status reports are communicated to the STAG
+representative of the project
+- Resolving conflict within the project and escalating to leadership
+when required
+- Outreach and recruitment for the project
+- Provide retrospective of a project to the STAG
+
```
Comment (lumjjb) [inline governance/roles.md:159]:
```suggestion

```

File: governance/roles.md [hunk 3]
```diff
@@ -140,6 +141,60 @@ Project Leads are nominated and approved by the following process:
 Depending on the expected length of the project, Project Leads may be assigned
 as OWNERS of the project directory in this repository to help facilitate and
  moderate new content to the project.
+ 
+### Duties of a project lead
+
+Project leads are responsible for the following:
```
Comment (lumjjb) [inline governance/roles.md:147]:
```suggestion
Project leads are responsible for the following:

```

**Review by lumjjb — COMMENTED** [review]
linter fixes

File: governance/roles.md [hunk 4]
```diff
@@ -140,6 +141,60 @@ Project Leads are nominated and approved by the following process:
...
+representative of the project
+- Resolving conflict within the project and escalating to leadership
+when required
+- Outreach and recruitment for the project
+- Provide retrospective of a project to the STAG
+ 
+## Role of STAG representatives
+
+STAG representatives are leadership executive sponsors of a project. The
+STAG representative acts as a point of liason with the STAG leadership team
```
Comment (lumjjb) [inline governance/roles.md:162]:
```suggestion
STAG representative acts as a point of liaison with the STAG leadership team
```

File: governance/roles.md [hunk 5]
```diff
@@ -140,6 +141,60 @@ Project Leads are nominated and approved by the following process:
...
+This includes co-chairs, technical leads and co-chair emiritus roles.
+A STAG representative may be of capacity of an individual contributor but
+cannot be the project lead of the same project they are representing.
+
+STAG representatives are appointed through the following process:
+
+- A member of the leadership indicates that they would like to be a STAG
+representative on a project by:
+  - editing the issue's "STAG Representative" field to their github ID.
+  - setting the issue milestone to the corresponding mileston created for
```
Comment (lumjjb) [inline governance/roles.md:175]:
```suggestion
  - setting the issue milestone to the corresponding milestone created for
```

File: governance/roles.md [hunk 6]
```diff
@@ -140,6 +141,60 @@ Project Leads are nominated and approved by the following process:
...
+- Provide retrospective of a project to the STAG
+ 
+## Role of STAG representatives
+
+STAG representatives are leadership executive sponsors of a project. The
+STAG representative acts as a point of liason with the STAG leadership team
+and takes on a role of mentorship for the project lead(s).
+
+STAG representatives are required to be part of the STAG leadership team.
+This includes co-chairs, technical leads and co-chair emiritus roles.
```
Comment (lumjjb) [inline governance/roles.md:166]:
```suggestion
This includes co-chairs, technical leads and co-chair emeritus roles.
```

**Review by lumjjb — COMMENTED** [review]
spelling fix

**Pull request merged**